tsconfig.json

```diff
@@ -18,6 +18,7 @@
     "clients/**/*.js",
     "build/**/*.js",
     "./types/**/*.d.ts",
+    "*.js",
```

Comment:
what prompted this?

Reply:
type checking `./eslint-local-rules.js`. I can narrow it if it messes with people's local, untracked scripts :)

Reply:
yes please if we can :)